### PR TITLE
Feature/packer

### PR DESCRIPTION
**Pull request by natemellendorf**
### Checklist

- [X] All new jobs, commands, executors, parameters have descriptions
- [X] Examples have been added for any significant new features
- [X] README has been updated, if necessary

### CODE REVIEW

**Comment by natemellendorf** [comment]
This is ready for review.
Once this is patched into the Ansible repo, this will be ready for merge
https://github.com/netgaintechnology/ansible/pull/15

**Comment by natemellendorf** [comment]
Ok, Ansible fix was merged.
This is now ready for review and merge as well.
Here's a successful job run with this orb:
https://app.circleci.com/jobs/github/netgaintechnology/packer_win2016/243

If you wish to test, feel free to pull down this branch, update the readme, and then push it.
https://github.com/netgaintechnology/packer_win2016/tree/feature/ansible

File: src/executors/packer.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+description: |
+  Docker image for Packer on alpine linux
+
+resource_class: medium
+
+docker:
+  - image: alpine:latest
```
Comment (Stromweld) [inline src/executors/packer.yml:7]: see comments from build orb

File: src/jobs/packer_deploy.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        - run:
+            name: Clone Ansible Repo into ansible directory
+            command: |
+              mkdir ansible
+              git clone <<parameters.ansible_repo>> ansible
+
+  - run:
+      name: Update system packages
+      command: |
+        apk add --update git bash openssl curl openssh-client python py-boto py-dateutil py-httplib2 py-jinja2 py-paramiko py-pip py-setuptools py-yaml tar unzip
```
Comment (Stromweld) [inline src/jobs/packer_deploy.yml]: see build orb comments

Comment (natemellendorf) [reply src/jobs/packer_deploy.yml]: Done and done. Thanks!

**Comment by natemellendorf** [comment]
@Stromweld I made the requested changes.
I have the orb running and building out a new image now. You can review that with the workflow status here: https://app.circleci.com/jobs/github/netgaintechnology/packer_win2016/267
Outside of that, the corrections I made are outside the scope of the build process. This should be ready to merge now.

**Review by Stromweld — COMMENTED** [review]
looks good

**Pull request merged**